api/src/main/java/org/ehcache/exceptions/CacheWritingException.java

```diff
@@ -27,20 +27,36 @@
 public class CacheWritingException extends RuntimeException {
 
   private static final long serialVersionUID = -3547750364991417531L;
-
-  CacheWritingException() {
+  
+  /**
```

Comment:
While this will please check style, it is not acceptable. Please add a basic sentence - see other exceptions.
Applies to all the constructors.